multiaccounts/settings/columns.go

```diff
@@ -392,6 +392,11 @@ var (
 		dBColumnName:   "test_networks_enabled",
 		valueHandler:   BoolHandler,
 	}
+	IsSepoliaEnabled = SettingField{
```

Comment:
@alaibe do we want/plan to sync this across devices or that's just a local setting?

Reply:
Just local setting!